CHANGELOG.rst

```diff
@@ -3,6 +3,16 @@ Changelog
 =========
 
 
+2.0.0 (unreleased)
+==================
+
+* Added support for Django 3.1
+* Dropped support for Python 2.7 and Python 3.4
+* Dropped support for Django < 2.2
+* Added support for sorl thumbnail
```

Comment:
i could not find the code for this and the CORS fix? can you pls point me to it?

Reply:
@kinkerl that was missing from the changelog in general, PR was https://github.com/divio/djangocms-bootstrap4/pull/108

Reply:
kk